constellations/gemini-admin/src/ask_for_help/afhEdit.sass

```diff
@@ -0,0 +1,50 @@
+@import "../_.constants.sass"
+
+#view-afh-edit
+  width: $spacing96x
+  padding: $spacing1x $spacing8x $spacing4x
+  box-sizing: border-box
+
+  & h2
+    margin: $spacing4x 0
```

Comment:
The indents in this file look really small. Not sure if it's a sass thing, but it should be "one indent=4 spaces" typically